cli/src/action/database.rs

```diff
@@ -18,7 +18,7 @@ use super::Action;
 use crate::error::CliError;
 use diesel::{connection::Connection as _, pg::PgConnection};
 #[cfg(feature = "database-migrate-biome")]
-use splinter::biome::datastore::run_postgres_migrations;
```

Comment:
Typo in commit message "databasestore -> datastore"

Reply:
Updated